templates/subscription/show.html.twig

```diff
@@ -138,6 +138,8 @@
                                             {{ _self.badge('Detached', 'fill-gray-500') }}
                                         {% elseif subscription.status.value == 'error' %}
                                             {{ _self.badge('Error', 'fill-red-500') }}
+                                        {% elseif subscription.status.value == 'failed' %}
+                                            {{ _self.badge('Failed', 'fill-red-500') }}
                                         {% endif %}
```

Comment:
Maybe we should add a unkown case here too, just to be safe that we can identify it easily if we add another case. WDYT?

Reply:
I added a fallback